src/lib/storage/chunk.hpp

```diff
@@ -151,6 +152,19 @@ class Chunk : private Noncopyable {
    */
   size_t estimate_memory_usage() const;
 
+  void increase_invalid_row_count(uint64_t count) const;
```

Comment:
can you doc this function and the function below (briefly?) as to what purpose they serve?